CMakeLists.txt

```
@@ -70,11 +72,13 @@ add_library(roboteam_utils STATIC
 target_include_directories(roboteam_utils
         INTERFACE include
         PRIVATE include/roboteam_utils
-        PRIVATE src)
+        PRIVATE src
+        )
 
 target_link_libraries(roboteam_utils
-        PUBLIC roboteam_proto
+        PUBLIC networking
```

You shouldn't need the publishers and subscribers here, you only need the proto definitions.

---

Why is this not roboteam_proto?

---

Did this when i was testing, you're right.
```suggestion
        PUBLIC roboteam_proto
```